src/reducers/versions.tsx

```diff
@@ -153,10 +153,13 @@ type InternalVersionFile = {
   content: string;
   created: string;
   downloadURL: string | null;
+  filename: string;
   id: number;
   isMinified: boolean;
-  sha256: string | null;
+  mimeType: string;
+  sha256: string;
```

Comment:
it can't be `null` any longer?

Reply:
Correct. When `sha256` was in `entries` it could be `null`, but in `file` it should never be `null`.